### PR TITLE
initial RSA post

### PR DESCRIPTION
**Pull request by alisaduncan**
This blog post has the following features:

- [x] A title that's approved by Dev Advocacy
- [ ] A URL approved by Dev Advocacy
- [x] The content has been run through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist
- [ ] Images are compressed appropriately
- [ ] Social image previews well on Twitter and LinkedIn
- [ ] Review request for editorial/grammar/clarity (developer advocate or Colton)

### CODE REVIEW

File: _source/_posts/2024-05-01-okta-rsa.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The Shared Signals Framework (SSF) is a huge step in enabling formerly siloed applications to see the whole picture of whether a user's behavior looks normal or suspicious. SSF is developed by the OpenID Foundation, which is also the home of standards like OIDC, FAPI, and MODRNA. 
+
+Although any software can send and receive these signals, Okta is uniquely positioned to work with our integrators to make SSF easy for identity administrators to use. 
+
+To learn more about the Shared Signals Framework, check out the OpenID foundation's [SSF Working Group](https://openid.net/specs/openid-sharedsignals-framework-1_0.html) or the explainer site [sharedsignals.guide](https://sharedsignals.guide/). Learn about how Okta and our integrators are deploying some of the world's first SSF implementations on [our developer podcast](TODO FIXME LINK)
+
+> Abstract on SSF filled out by E. Dunham. Also please replace link to podcast.
```
Comment (edunham) [inline _source/_posts/2024-05-01-okta-rsa.md]:
```suggestion
```

File: _source/_posts/2024-05-01-okta-rsa.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+> Link to blog pointing to podcast on SSF with Jamf
```
Comment (edunham) [inline _source/_posts/2024-05-01-okta-rsa.md]:
```suggestion
```
podcast recording is happening today so I'll add the link to this post when I publish its post

File: _source/_posts/2024-05-01-okta-rsa.md (hunk 1 above)
Comment (edunham) [reply _source/_posts/2024-05-01-okta-rsa.md]: the above text is the abstract that I wrote in the google doc last week; is there more needed?

File: _source/_posts/2024-05-01-okta-rsa.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Universal Logout
+What should happen next when suspicious activity is detected on a user's account? Identity providers can easily prevent a user from starting new sessions, but in the past, there hasn't been a good way to end all of a user's existing sessions. Nobody wants to re-authenticate for every task they perform, but in the past, these productivity and convenience features have given attackers an avenue to impersonate legitimate users by stealing session information. However, it's harder for an attacker to create a session than to continue a stolen session. 
+
+Universal Logout addresses this problem by offering administrators and their automation a way to log users out of all their existing sessions with an identity provider. UL pairs with SSF and other intrusion detection tooling to get the best of both worlds: Users can keep the productivity benefits of long sessions and only do extra re-authentication when threats are detected, yet intruders experience the challenges of session termination immediately when they try to misuse ill-gotten access. 
+
+> Abstract on UL filled out by Semona Igama
```
Comment (edunham) [inline _source/_posts/2024-05-01-okta-rsa.md]:
```suggestion
```

the above text *is* the abstract; we just need approval of it by @isemona

File: _source/_posts/2024-05-01-okta-rsa.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+> Link to blog post on UL
+> Invitation for ISVs to contact us at dev-advocacy@okta.com if interested in testing w/Okta in a pilot program
```
Comment (edunham) [inline _source/_posts/2024-05-01-okta-rsa.md]:
```suggestion
If you would like to join a pilot program of integrators working with SSF and UL, contact us at dev-advocacy@okta.com.
```

**Review by edunham — APPROVED** [review]

File: _source/_posts/2024-05-01-okta-rsa.md (hunk 2 above)
Comment (alisaduncan) [reply _source/_posts/2024-05-01-okta-rsa.md]: @edunham Would you mind changing this text then to remove references to linking? we shouldn't have a bad link.

File: _source/_posts/2024-05-01-okta-rsa.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+Universal Logout addresses this problem by offering administrators and their automation a way to log users out of all their existing sessions with an identity provider. UL pairs with SSF and other intrusion detection tooling to get the best of both worlds: Users can keep the productivity benefits of long sessions and only do extra re-authentication when threats are detected, yet intruders experience the challenges of session termination immediately when they try to misuse stolen access. 
+
+While the [Universal Logout](https://developer.okta.com/docs/guides/oin-universal-logout-overview/) feature provided by Okta has not yet been released, you can build the endpoint that IdPs such as Okta or threat detection tools can call to end a user session ASAP by following the instructions on this blog: [How to Instantly Sign a User Out across All Your Apps](https://developer.okta.com/blog/2024/04/30/express-universal-logout).
```
Comment (alisaduncan) [inline _source/_posts/2024-05-01-okta-rsa.md]:
```suggestion
While the [Universal Logout](https://developer.okta.com/docs/guides/oin-universal-logout-overview/) feature provided by Okta has not yet been released, you can build the endpoint that IdPs such as Okta or threat detection tools can call to end a user session ASAP by following the instructions on this blog: [How to Instantly Sign a User Out across All Your Apps](/blog/2024/04/30/express-universal-logout).
```

File: _source/_posts/2024-05-01-okta-rsa.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+If you would like to join a pilot program of integrators working with SSF and UL, contact us at dev-advocacy@okta.com.
```
Comment (alisaduncan) [inline _source/_posts/2024-05-01-okta-rsa.md]:
```suggestion
If you want to join a pilot program of integrators working with SSF and UL, contact us at wic-dev-advocacy@okta.com.
```

File: _source/_posts/2024-05-01-okta-rsa.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: 
+type: awareness
+---
+
+We are excited to have a presence at RSA Conference 2024! On Tuesday, May 7th, we will speak on [Identity: Your key to stop breaches](https://www.rsaconference.com/usa/agenda/session/Okta%20Identity%20Your%20key%20to%20stop%20breaches). Consider why you don't want to miss that session:
```
Comment (alisaduncan) [inline _source/_posts/2024-05-01-okta-rsa.md]:
```suggestion
We are excited to have a presence at RSA Conference 2024! On Tuesday, May 7th, we will speak on [Identity: Your key to stop breaches](https://www.rsaconference.com/usa/agenda/session/Okta%20Identity%20Your%20key%20to%20stop%20breaches). Consider why you don't want to miss this session: 
```

File: _source/_posts/2024-05-01-okta-rsa.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+The Shared Signals Framework (SSF) is a huge step in enabling formerly siloed applications to see the whole picture of whether a user's behavior looks normal or suspicious. SSF is developed by the OpenID Foundation, which is also the home of standards like OIDC, FAPI, and MODRNA. 
+
+Although any software can send and receive these signals, Okta is uniquely positioned to work with our integrators to make SSF easy for identity administrators to use. 
+
+To learn more about the Shared Signals Framework, check out the OpenID foundation's [SSF Working Group](https://openid.net/specs/openid-sharedsignals-framework-1_0.html) or the explainer site [sharedsignals.guide](https://sharedsignals.guide/). Learn about how Okta and our integrators are deploying some of the world's first SSF implementations on [our developer podcast](TODO FIXME LINK)
```
Comment (edunham) [inline _source/_posts/2024-05-01-okta-rsa.md]:
```suggestion
To learn more about the Shared Signals Framework, check out the OpenID foundation's [SSF Working Group](https://openid.net/specs/openid-sharedsignals-framework-1_0.html) or the explainer site [sharedsignals.guide](https://sharedsignals.guide/). 
```

**Pull request merged**